### PR TITLE
zh-translation:/docs/ops/diagnostic-tools/component-logging/index.md

### PR DESCRIPTION
**Pull request by ghl1024**
ref: [servicemesher/istio-official-translation/issues/1242](https://github.com/servicemesher/istio-official-translation/issues/1242)

### CODE REVIEW

**Comment by yuxiaobo96** [comment]
@rootsongjc 
Please help merge.

File: content/zh/docs/ops/diagnostic-tools/component-logging/index.md [hunk 1]
```diff
@@ -1,74 +1,59 @@
 ---
-title: Component Logging
-description: Describes how to use component-level logging to get insights into a running component's behavior.
+title: 组件记录
```
Comment (rootsongjc) [inline content/zh/docs/ops/diagnostic-tools/component-logging/index.md]: 组件记录改成组件日志记录吧

Comment (ghl1024) [reply content/zh/docs/ops/diagnostic-tools/component-logging/index.md]: @rootsongjc
OK，已修改

**Pull request merged**